build-properties.xml

```diff
@@ -47,5 +47,7 @@
     <name>net.sf.saxon.BasicTransformerFactory</name>
     <location>common${file.separator}lib${file.separator}saxon</location>
   </processor>
-  
+
+  <uvepssDir>common/lib/staticSearch</uvepssDir>
```

Comment:
As mentioned before, it is unnecessary to specify `\` instead of `/` to get Ant to recognize path components in Windows. (Except, it turns out, for the `@to` and `@from` attributes of mappers — I think, but do not know, that this is because they are considered _strings_, not _paths_ nor _URLs_.) As evidence:

- StaticSearch, the TEI P5 builds, and the TEI Stylehsheets builds work fine on Windows, none of which use `${file.separator}`.
- “A forward slash will work as a directory delimiter even on Windows systems; Ant is smart enough to know what you mean.” Holzner, Steve, _Ant, The Definitive Guide_ 2nd Ed. p. 18. O’Reilly, 2005.
- “The rules for handling path names are that 'DOS-like pathnames are handled', 'Unix like paths are handled'. Disk drives -'C:'- are handled on DOS-based boxes, but placing them in the build.xml file ruins all chances of portability. Relative file paths are much more portable. ” Loughran, Steve, _[Apache Ant™ in Anger](https://ant.apache.org/ant_in_anger.html):
Using Apache Ant in a Production Development System_. Last updated 2005-03-16.

Thus I would much prefer the reverse, to use the `${file.separator}` syntax only when it is needed (on mappers) or at most when it is needed (on mappers) and when it might confuse a hard core Windows user (in messages) — although my experience has been that many Windows users do not even really see those two characters as different.

Reply:
Interesting! I didn't know that. Objection withdrawn.